src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -583,9 +686,41 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
               </AccordionPanel>
             </AccordionItem>
           )}
+          {userContext.apiType === "SQL" &&
+            userContext.authType === AuthType.AAD &&
+            configContext.platform !== Platform.Fabric && (
```

Comment:
Let's use `isFabric()`.
```suggestion
            !isFabric() && (
```

Reply:
Adding to latest commit with import.  Will resolve when committed.